### DISCUSSION TITLE
Question about new relay service

### DISCUSSION THREAD

**Discussion by PathfinderNetworks** [Ylianst/MeshCentral#4231] [General]
I'm configuring my MeshCentral server for the new relay through agent service.  I am running 1.0.46.  I just watched the new YouTube video about this feature but have one question that wasn't answered.  
I have created the 'No Agent devices relayed thru agent' device group. But how do I add the devices I want to access through the relay?  When I click the 'Add Device' link I don't have the options I believe I should have to correctly add a device.  All I see is the 'Add local device' popup that has an entry for Device Name, Hostname, and Type.  Under Type the only options are 'Windows (RDP)', Linux (SSH/SCP/VNC), and 'macOS (SSH/SCP/VNC).  
I'm thinking I should have more options under Type?

**Comment by Ylianst** [comment]
When adding a new local device, the "type" only serves to indicate what types of features you want enabled with this device.

- If you select "Windows (RDP)" you will get the RDP feature on the desktop tab along with HTTP and HTTPS web relay links.
- If you select "Linux (SSH/SCP/VNC)" you get SSH on the terminal tab, SCP on the files tab and also get HTTP and HTTPS web relay links.

Your type choice does not even have to be accurate. I could have a "All Services" type, or even have check boxes to select what services you want.

Let me know if that makes sense.

**Reply by PathfinderNetworks** [reply]
That does make sense.  I'll get more devices set up this week and do some more testing.  I'm used to using the Mesh Router for this function- and it works great. But it will be very nice to be able to have often accessed devices be available through this web relay option.
